homeworks/homework-1/app/src/components/CourseListItem.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import {
+	Button,
+	Grid,
+	Row,
+	Col
+} from "react-bootstrap";
+import PropTypes from "prop-types";
+
+class CourseListItem extends React.Component {
+
```

Comment:
Please specify used props via PropTypes, and check it across all files, because i believe i already note about proptypes. You can read here about proptypes: https://reactjs.org/docs/typechecking-with-proptypes.html